pkg/util/validation/limits.go

```diff
@@ -68,6 +68,8 @@ type Limits struct {
 	MaxLocalMetadataPerMetric           int `yaml:"max_metadata_per_metric" json:"max_metadata_per_metric"`
 	MaxGlobalMetricsWithMetadataPerUser int `yaml:"max_global_metadata_per_user" json:"max_global_metadata_per_user"`
 	MaxGlobalMetadataPerMetric          int `yaml:"max_global_metadata_per_metric" json:"max_global_metadata_per_metric"`
+	// Snapshot
+	MemorySnapshotOnShutdownPerUser bool `yaml:"memory_snapshot_on_shutdown_per_user" json:"memory_snapshot_on_shutdown_per_user"`
```

Comment:
It is a little bit weird to have `_per_user` suffix since all limits here are per user. For those old limits we can keep them for now but let's remove `_per_user` for this new limit.

Reply:
```
	MaxLocalSeriesPerUser    int `yaml:"max_series_per_user" json:"max_series_per_user"`
	MaxGlobalSeriesPerUser   int `yaml:"max_global_series_per_user" json:"max_global_series_per_user"`
	MaxGlobalMetricsWithMetadataPerUser int `yaml:"max_global_metadata_per_user" json:"max_global_metadata_per_user"`
```

Drawing on the configuration that already exists, I prefer to be consistent. Of course, you can also submit a PR later to fix it.

Reply:
As I mentioned, I think that's a tech debt we had. Let's remove the suffix for the new limit.